app/views/local_petitions/constituency_lookup_failed.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Sorry!</h1>
```

Comment:
If this is a page on it's own you could add a class of page-title to the h1 to hook into some existing styles.